packages/neuron-wallet/src/services/cells.ts

```diff
@@ -438,14 +458,64 @@ export default class CellsService {
     return cellEntity
   }
 
+  public static getLiveCells = async (walletId: string) => {
+    const hdPublicKeyInfos = await getConnection()
+      .getRepository(HdPublicKeyInfo)
+      .createQueryBuilder()
+      .where({ walletId })
+      .getMany()
+    const blake160s = hdPublicKeyInfos.map(v => v.publicKeyInBlake160)
+    const multisigArgs = hdPublicKeyInfos.map(v => Multisig.hash([v.publicKeyInBlake160]))
+    // find all outputs except cheque
+    const outputs = await getConnection()
+      .getRepository(OutputEntity)
+      .createQueryBuilder('output')
+      .leftJoinAndSelect('output.transaction', 'tx')
+      .where({
+        status: In([OutputStatus.Live, OutputStatus.Sent]),
```

Comment:
I believe it will become a live cell after the transaction broadcast success.

Reply:
I found a similar method at https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/services/cells.ts#L441

We can keep their name consistent, or simply extend the `getLiveOrSentCellByWalletId` method

Reply:
Any update on this comment

Reply:
I'm working on another issue, I will check and update after that.

Reply:
There are some differences between these:
`getLiveCell` is getting a live cell from `outPoint`, and `getLiveCells` is getting all live cells from a wallet.
`getLiveOrSentCellByWalletId` will limit `lockCodeHash` and `lockHashType` and skip cell with data, and `getLiveCells` has no limit.

Maybe it's better to rename `getLiveCell` to `getLiveCellByOutPoint`, and rename `getLiveOrSentCellByWalletId` to `getLiveCellByWalletAndLockCode`? What do you think?

Reply:
I would suggest extending `getLiveOrSentCellByWalletId` to accept a filter, and it should return all cells if the filter is empty.

> Maybe it's better to rename `getLiveCell` to `getLiveCellByOutPoint`

I think `getLiveCell(outPoint)` is clear enough because `outPoint` is specified in the parameters.

Reply:
updated